pkg/extension/workspace/example/envoy/util/load_test.go

```diff
@@ -68,28 +65,11 @@ func TestNewFakeAnyResolver(t *testing.T) {
 		test := test // pin! see https://github.com/kyoh86/scopelint for why
 
 		t.Run(test.name, func(t *testing.T) {
-			resolver := newFakeAnyResolver()
+			resolver := fakeAnyResolver{}
 
-			actual, err := resolver.Resolve(test.typeURL)
+			actual, err := resolver.FindMessageByURL(test.typeURL)
 			require.NoError(t, err)
-
-			props := proto.GetProperties(reflect.ValueOf(actual).Elem().Type())
```

Comment:
this was library specific. the key here is that it doesn't err. other tests show that "any" values are consumable